### ISSUE TITLE
Add a troubleshooting entry for no monitor display

### ISSUE THREAD

**Issue by itsderek23** [luxonis/depthai-docs-website#14]
Initial draft:

## Nothing is displayed when the RPi Compute is connected to a monitor.

The RPi may be unable to detect a connected monitor if it is in power-save mode. When this occurs, the RPi enters headless mode (no HDMI drive). Ensure the monitor is awake and active when the RPi is booting so it can detect the monitor.

/cc @Luxonis-Brandon

## Background

[See this Slack Community thread](https://luxonis-community.slack.com/archives/CT79Z6GP5/p1582902208029800).

<img width="871" alt="image" src="https://user-images.githubusercontent.com/7880/75561708-b4ecac80-5a04-11ea-8904-d6289ec5a42d.png">

**Comment by Luxonis-Brandon** [comment]
LGTM.  So one note on a technical side: I'm not sure if perhaps there exists a setting in Raspberry Pi that like 'force assumes' a monitor is present.  May be worth googling at some point.

**Comment by itsderek23** [comment]
> I'm not sure if perhaps there exists a setting in Raspberry Pi that like 'force assumes' a monitor is present.

Good point! [Via this SO question](https://raspberrypi.stackexchange.com/questions/2169/how-do-i-force-the-raspberry-pi-to-turn-on-hdmi):



Add these two lines to /boot/config.txt and reboot Raspbmc:

```
hdmi_force_hotplug=1
hdmi_drive=2
```
hdmi_force_hotplug=1 sets the Raspbmc to use HDMI mode even if no HDMI monitor is detected. hdmi_drive=2 sets the Raspbmc to normal HDMI mode (Sound will be sent if supported and enabled). Without this line, the Raspbmc would switch to DVI (with no audio) mode by default.

**Comment by Luxonis-Brandon** [comment]
Sweet, thanks.  So we'll add this to new default Raspberry Pi images for next shipments.  CC: @Luxonis-Brian .  

Still good to have this on the site - particularly in the future so that folks can disable it if they want to save power/etc.

**Comment by VanDavv** [comment]
I'll add this section, would definitely be helpful when someone instead of a working product would see just a blank screen - very frustrating. Assigning myself to keep track, will circle back with PR
